libs/smart-ngrx/src/actions/action.service/update.class.ts

```
@@ -125,8 +85,112 @@ export class Update<T extends SmartNgRXRowBase> {
-    return of([originalRow]);
+    return of([originalRow as T]);
+  }
+
+  /**
+   * Creates the update pipeline for processing row updates
+   *
+   * @returns An Observable that processes and handles row updates
+   */
+  private createUpdatePipeline(): Observable<void> {
+    function mapToVoidFn(): void {
+      return undefined;
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)